backoffice/src/Views/Email/Index.cshtml

```diff
@@ -1,7 +1,42 @@
+@model EmailListView
 @{
     ViewData["Title"] = "Emails";
 }
 
 <h4 class="display-8">@ViewData["Title"]</h4>
 
+<table class="table">
+    <thead>
+    <tr>
+        <th scope="col">Recipient</th>
+        <th scope="col">Subject</th>
+        <th scope="col">Type</th>
+        <th scope="col">Last Modified</th>
+        <th scope="col">Is Sent?</th>
+        <th scope="col">Actions</th>
+    </tr>
+    </thead>
+    <tbody>
+    @foreach (var email in Model.EmailList)
+    {
+        <tr>
+            <th scope="row">
+                <a asp-action="GetPreview" asp-route-id="@email.Id">@email.Recipient</a>
+            </th>
+            <td>@email.Subject</td>
+            <td>@email.TypeOfEmail.ToString()</td>
+            <td>@email.Date.ToString(format:"MM/dd/yyyy")</td>
```

Comment:
Should be able to set this on the class itself 

See : https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.displayformatattribute.dataformatstring?view=net-7.0

Reply:
moved into the constructor of the EmailView model